### PR TITLE
Update tencent

### PR DESCRIPTION
**Pull request by Lablace**
ref: the last section of https://www.wxopen.club/topic/62375e6f8f48e575a4cc6f23

### CODE REVIEW

**Comment by Lablace** [comment]
Also this: https://www.whois.com/whois/weixinbridge.com , which shows the owner is Tencent.

File: data/tencent
Comment (rootmelo92118) [inline data/tencent:1]: Sort it

**Review by rootmelo92118 — CHANGES_REQUESTED** [review]

**Comment by Lablace** [comment]
Had it sorted. Not very familiar with the resolve routine, sorry if it looks like a mess.

**Comment by Lablace** [comment]
Sure, had it moved.

**Review by rootmelo92118 — APPROVED** [review]

**Pull request merged**